### ISSUE TITLE
Reactjs issue 1

### ISSUE THREAD

**Issue by hoanghiep** [nikhilkalige/docblockr#195]
```
var QuitJob = React.createClass({
  
});
```
Here is my reactjs code, when i type /** to gen docblockr, an error occurs

```
TypeError: Cannot read property 'value' of null
    at DocsParser.JsParser.get_arg_type (C:\Users\xxx\.atom\packages\docblockr\lib\languages\javascript.js:81:15)
    at DocsParser.parse_args (C:\Users\xxx\.atom\packages\docblockr\lib\docsparser.js:265:24)
    at DocsParser.format_function (C:\Users\xxx\.atom\packages\docblockr\lib\docsparser.js:130:32)
    at DocsParser.parse (C:\Users\xxx\.atom\packages\docblockr\lib\docsparser.js:44:37)
    at DocBlockrAtom.module.exports.DocBlockrAtom.parse_command (C:\Users\xxx\.atom\packages\docblockr\lib\docblockr-worker.js:287:29)
    at atom-workspace.<anonymous> (C:\Users\xxx\.atom\packages\docblockr\lib\docblockr-worker.js:77:16)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\xxx\AppData\Local\atom\app-1.9.1\resources\app.asar\src\command-registry.js:260:29)
    at C:\Users\xxx\AppData\Local\atom\app-1.9.1\resources\app.asar\src\command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\xxx\AppData\Local\atom\app-1.9.1\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\xxx\AppData\Local\atom\app-1.9.1\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (C:\Users\xxx\AppData\Local\atom\app-1.9.1\resources\app.asar\src\window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (C:\Users\xxx\AppData\Local\atom\app-1.9.1\resources\app.asar\src\window-event-handler.js:3:61)
```

**Comment by MoritzKn** [comment]
At its core this is the same problem as #196: `JsParser.prototype.parse_function` applies instead of `JsParser.prototype.parse_var`. This is because `parse_function('var foo = bar.foo({});')` returns `["foo", "{}", null]` instead of `null`, what I would expect.
The regex in question: [`'(?:(?P<name2>' + this.settings.fnIdentifier + '))?'`](https://github.com/nikhilkalige/docblockr/blob/master/lib/languages/javascript.js#L37)

**Comment by MoritzKn** [comment]
This was fixed with #199. And shouldn't happen anymore in version 0.8.0.
